libraries/AP_GPS/AP_GPS_ExternalAHRS.cpp

```diff
@@ -82,6 +82,17 @@ void AP_GPS_ExternalAHRS::handle_external(const AP_ExternalAHRS::gps_data_messag
 
     state.last_gps_time_ms = AP_HAL::millis();
 
+    if (pkt.have_heading) {
```

Comment:
Isn't your heading going to get stuck when this becomes false?

Also, really need a user before we could merge this.